wiki/math/registration-techniques.md

```diff
@@ -0,0 +1,132 @@
+# Registration Techniques in Robotics
```

Comment:
I'd love to get this into the wiki. However, it doesn't follow any of our style guidelines and Jekyll is finicky as it is. Please [review the template](https://raw.githubusercontent.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/d595f7aac3f0000ce40d845113940171a4f8ff1a/_templates/template.md) and update this accordingly